site/docs/resources/faq.md

```diff
@@ -26,6 +27,42 @@ No, it's not.
    As always, TypeScript will auto-complete the parameter names for you.
 3. Perhaps double-check the method signature for [actions](../guide/context#available-actions) on `ctx` [here](https://deno.land/x/grammy/mod.ts?s=Context#Methods), or for API methods (`ctx.api`, `bot.api`) [here](https://deno.land/x/grammy/mod.ts?s=Api#Methods).
 
+## How Can I Access the Chat History?
+
+You can't.
+
+Telegram does not store the messages for your bot.
```

Comment:
That as it is phrased right now, is right on the high level, but also a lie if you take it seriously. Would you like to go with it?